onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -2157,6 +2158,10 @@ def _where_input_wrangler(
         torch.float32,
         torch.float16,
     ),
+    "aten.stft": (
+        torch.float32,
+        # torch.float16,  RuntimeError: MKL FFT doesn't support tensors of type: Half
```

Comment:
Please add an xfail for this for now. We will consolidate the lists later. Thanks!

Reply:
Done